portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro

```diff
@@ -331,7 +331,7 @@ definition {
 		MenuItem.click(menuItem = "Delete");
 	}
 
-	macro duplicateFieldsets {
```

Comment:
After a rebase code didn't pass Poshi validation since other tests are using duplicateFieldsets macro. Please be careful when modifying or deleting common methods